MANIFEST.in

```diff
@@ -1,2 +1,4 @@
 include requirements.txt
-include CHANGELOG.rst
\ No newline at end of file
+include CHANGELOG.rst
+include lightning.qubit.toml
```

Comment:
Should we add one for L-GPU while we are at it?

Reply:
I think it is not needed at the moment since it is not compatible with Catalyst. I would prefer to add it once we have the compatibility.